### PR TITLE
Sample code change for Routing/Account Number

### PR DESCRIPTION
**Pull request by saikatbasu01**
Random generation of Account Number logic added as DebitBankAccount sample code starts failing if the same account number-routing number combination is used multiples times.

### CODE REVIEW

File: CustomerProfiles/CreateCustomerProfile.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public static ANetApiResponse Run(string ApiLoginID, string ApiTransactionKey, s
             var creditCard = new creditCardType
             {
                 cardNumber      = "4111111111111111",
-                expirationDate  = "0718"
+                expirationDate  = "0719"
```
Comment (ashtru) [inline CustomerProfiles/CreateCustomerProfile.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: CustomerProfiles/UpdateCustomerPaymentProfile.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey, s
             var creditCard = new creditCardType
             {
                 cardNumber = "4111111111111111",
-                expirationDate = "0718"
+                expirationDate = "0719"
```
Comment (ashtru) [inline CustomerProfiles/UpdateCustomerPaymentProfile.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: CustomerProfiles/UpdateCustomerShippingAddress.cs [hunk 3]
```diff
@@ -27,7 +27,7 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey, s
             var creditCard = new creditCardType
              {
                  cardNumber = "4111111111111111",
-                 expirationDate = "0718"
+                 expirationDate = "0719"
```
Comment (ashtru) [inline CustomerProfiles/UpdateCustomerShippingAddress.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: PaymentTransactions/AuthorizeCreditCard.cs [hunk 4]
```diff
@@ -27,7 +27,7 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey, d
             var creditCard = new creditCardType
             {
                 cardNumber      = "4111111111111111",
-                expirationDate  = "0718"
+                expirationDate  = "0719"
```
Comment (ashtru) [inline PaymentTransactions/AuthorizeCreditCard.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: PaymentTransactions/ChargeTokenizedCreditCard.cs [hunk 5]
```diff
@@ -28,7 +28,7 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey)
             var creditCard = new creditCardType
             {
                 cardNumber = "4111111111111111",
-                expirationDate = "0718",
+                expirationDate = "0719",
```
Comment (ashtru) [inline PaymentTransactions/ChargeTokenizedCreditCard.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: PaymentTransactions/DebitBankAccount.cs [hunk 6]
```diff
@@ -24,11 +24,14 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey, d
                 Item = ApiTransactionKey
             };
 
+            Random rand = new Random(); 
+            int randomAccountNumber = rand.Next(100000000, int.MaxValue);
```
Comment (ashtru) [inline PaymentTransactions/DebitBankAccount.cs]: Starting can be 5-digit number. 
Test and verify locally once for 10000.

File: RecurringBilling/CreateSubscription.cs [hunk 7]
```diff
@@ -40,7 +40,7 @@ public static ANetApiResponse Run(String ApiLoginID, String ApiTransactionKey, s
             var creditCard = new creditCardType
             {
                 cardNumber      = "4111111111111111",
-                expirationDate  = "0718"
+                expirationDate  = "0719"
```
Comment (ashtru) [inline RecurringBilling/CreateSubscription.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: RecurringBilling/CreateSubscriptionFromCustomerProfile.cs [hunk 8]
```diff
@@ -41,7 +41,7 @@ public class CreateSubscriptionFromCustomerProfile
             var creditCard = new creditCardType
             {
                 cardNumber      = "4111111111111111",
-                expirationDate  = "0718"
+                expirationDate  = "0719"
```
Comment (ashtru) [inline RecurringBilling/CreateSubscriptionFromCustomerProfile.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

File: RecurringBilling/UpdateSubscription.cs [hunk 9]
```diff
@@ -32,7 +32,7 @@ public static ANetApiResponse Run(string ApiLoginID, string ApiTransactionKey, s
             var creditCard = new creditCardType
             {
                 cardNumber = "4111111111111111",
-                expirationDate = "0718"
+                expirationDate = "0719"
```
Comment (ashtru) [inline RecurringBilling/UpdateSubscription.cs]: Change it to 0728. Wherever you see 2018, increased it by 10 years.

**Review by ashtru — CHANGES_REQUESTED** [review]

**Comment by saikatbasu01** [comment]
Changes are modified as per review comments.

**Comment by ashtru** [comment]
@saikatbasu01 
Good work on making the changes!
Just take care of whitespaces changes next time. I did a follow-up [commit ](https://github.com/AuthorizeNet/sample-code-csharp/commit/ca47e558071c84adfd004041b38be6ab3d2a1673)this time.

**Pull request merged**